### PR TITLE
Make meowbot be able to send picture through chat

### LINKED ISSUE #3: Add random cat pic command
### What is the feature to be added?

Add a command that, when invoked, has MeowBot post a random cat picture.
(Could maybe instead do a "daily cat pic"; I bet there is an api for this.)

### Why?

Braden and Helena requested this 🤷‍♀️

### PR DESCRIPTION
**Pull request by shellychw**
use `m!pic` to query a picture from the cat api

- added CatApi client to send http request to `https://docs.thecatapi.com/`
- added new CatPictureListener to send a picture whenever `m!pic` is called

@lwise you need to generate an API key from `https://docs.thecatapi.com/` or I can send you mine.

Resolved #3

### CODE REVIEW

**Comment by shellychw** [comment]
Also need to add line

`CAT_API_KEY=<YOUR KEY>` to the `.env` file

File: src/main/kotlin/com/lwise/text/generators/MeowMessageGenerator.kt [hunk 1]
```diff
@@ -2,26 +2,27 @@ package com.lwise.text.generators
...
 
     val meowMessage = listOf<String>("meow", "nya", "miaou", "miao", "mew", "miau", "miaow", "mnou")
-    val preMessage = listOf<String>("...", "~~~", ":3 ")
+    val preMessage = listOf<String>("...", "~~~~ ", "<3 ")
     val postMessage = listOf<String>(
         "  (づ｡◕‿‿◕｡)づ", " ≧◡≦", "!!", "!", "...", " /ᐠ｡ꞈ｡ᐟ\\\\ ", "  /ᐠ｡‸｡ᐟ\\\\ ", " /ᐠ｡ꞈ｡ᐟ✿\\\\ ",
         " /ᐠ｡ꞈ｡ᐟ❁\\\\∫", " /ᐠ｡▿｡ᐟ\\\\ *ᵖᵘʳʳ*", " /ᐠ｡ﻌ｡ᐟ\\\\", " /ᐠ.ꞈ.ᐟ\\\\", " ✧/ᐠ-ꞈ-ᐟ\\\\ ", " (^..^)ﾉ",
-        " /ᐠ. ⱉ .ᐟ\\\\ﾉ   ", " /ᐠ｡ⱉ｡ᐟ\\\\ﾉ ᶠᵉᵉᵈ ᵐᵉ", "~~~", "!!!", ".......", "!", "~~~"
+        " /ᐠ. ⱉ .ᐟ\\\\ﾉ   ", " /ᐠ｡ⱉ｡ᐟ\\\\ﾉ ᶠᵉᵉᵈ ᵐᵉ", "~~~", "!!!", ".......", "!", "~~~", "/ᐠ｡ퟑ｡ᐟ\\\\",
+        " <3 ", "  \\\\/ᐠ-ᆽ-ᐟ \\\\ "
```
Comment (lwise) [inline src/main/kotlin/com/lwise/text/generators/MeowMessageGenerator.kt:13]: sneaky sneaky; you should put back `:3` too though

File: src/main/kotlin/com/lwise/util/CatApiClient.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import org.json.JSONArray
+import java.io.IOException
+
+object CatApiClient {
+
+    val dotenv = dotenv {
+        ignoreIfMissing = true
+    }
+
+    val API_KEY = dotenv["DATABASE_URL"]
```
Comment (lwise) [inline src/main/kotlin/com/lwise/util/CatApiClient.kt]: Why is the `API_KEY` using `DATABASE_URL`?

Comment (shellychw) [reply src/main/kotlin/com/lwise/util/CatApiClient.kt]: because I goofed

Comment (shellychw) [reply src/main/kotlin/com/lwise/util/CatApiClient.kt]: fixed

File: src/main/kotlin/com/lwise/text/generators/MeowMessageGenerator.kt (hunk 1 above)
Comment (shellychw) [reply src/main/kotlin/com/lwise/text/generators/MeowMessageGenerator.kt:13]: opsss  ` :3 ` face is back

**Review by lwise — APPROVED** [review]

**Pull request merged**